Dockerfile.dapper

```diff
@@ -1,9 +1,7 @@
 FROM golang:1.12.12-alpine3.10
 
 RUN apk -U --no-cache add bash git gcc musl-dev docker vim less file curl wget ca-certificates jq linux-headers zlib-dev tar zip squashfs-tools npm coreutils \
-    python3 py3-pip python3-dev openssl-dev libffi-dev libseccomp libseccomp-dev make
-RUN pip3 install 'tox==3.6.0'
-RUN apk -U --no-cache --repository http://dl-3.alpinelinux.org/alpine/edge/main/ add sqlite-dev sqlite-static
```

Comment:
is python3 removal here related to dqlite?

Reply:
Nope, just saw that we didn't need them